Src/Framework/Jobs/Castep.py

```diff
@@ -225,13 +225,13 @@ def run_step(self, index):
         
         data = {"time": timeStep}
 
-        # Retrieve the positions multiplied by Units.Bohr*Units.Hartree/Units.hbar and save them as universe velocities
+        # Retrieve the velocities multiplied by Units.Bohr*Units.Hartree/Units.hbar and save them

```

Comment:
Check the unit of velocities and whether it is has been correctly read and saved

Reply:
Hi, I have kept all the units as they were originally, but since you asked; according to https://www.tcm.phy.cam.ac.uk/castep/MD/node13.html, all the variables in MD files are in Hartree units. Therefore,
- to convert positions, the given value has to be multiplied by the value of Bohr radius, which is what happens on line 151 `config[0:self["n_atoms"], :] *= Units.Bohr`;
- to convert velocities, the given values have to be multiplied by the value of Bohr radius and Hartree energy as well as divided by reduced Planck's constant, which is what happens on line 152: `config[self["n_atoms"]:2*self["n_atoms"], :] *= Units.Bohr*Units.Hartree/Units.hbar`;
- to convert forces, the given values have to be multiplied by the value of Hartree energy and divided by Bohr's radius, which is what happens on line 153: `config[2*self["n_atoms"]:3*self["n_atoms"], :] *= Units.Hartree/Units.Bohr`.

 Please let me know if there is any mistake anywhere!